lighthouse-core/test/results/sample_v2.json

```diff
@@ -13,26 +13,11 @@
   "audits": {
     "is-on-https": {
       "id": "is-on-https",
-      "title": "Does not use HTTPS",
+      "title": "Uses HTTPS",
       "description": "All sites should be protected with HTTPS, even ones that don't handle sensitive data. This includes avoiding [mixed content](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content), where some resources are loaded over HTTP despite the initial request being servedover HTTPS. HTTPS prevents intruders from tampering with or passively listening in on the communications between your app and your users, and is a prerequisite for HTTP/2 and many new web platform APIs. [Learn more](https://web.dev/is-on-https).",
-      "score": 0,
-      "scoreDisplayMode": "binary",
-      "displayValue": "1 insecure request found",
-      "details": {
-        "type": "table",
-        "headings": [
-          {
-            "key": "url",
-            "itemType": "url",
-            "text": "Insecure URL"
-          }
-        ],
-        "items": [
-          {
-            "url": "http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"
-          }
-        ]
-      }
+      "score": null,
+      "scoreDisplayMode": "error",
+      "errorMessage": "Required InspectorIssues gatherer did not run."
```

Comment:
hmmm

should we use optionalArtifacts again?
seems good.

but also we can add a stub to the artifacts fixtures

Reply:
No, I'll just fix the sample artifacts.

don't think optional artifacts fits here. it's meant to allow flexibility in  which gatherers can run in master today, based only on what gatherers are in the config. since InspectorIssues is in the default config, it won't benefit.